docs/foundations/validation/geospatialValidation.md

```diff
@@ -0,0 +1,103 @@
+# Verification and Validation of Geospatial Measurements
+
+Verification and validation of geospatial measurements on SAR imagery is an integral part of ICEYE's quality control process. Below you will find our latest validation report. More reports will be added as they become available.
+
+## Method
+'*Geolocation*' is the name of the process used to find the univocal correspondence between image pixels and their position on the Earth's surface. In this section we introduce a geolocation algorithm that we use to measure the geospatial accuracy of ICEYE imagery. 
+
+The purpose of geolocation is to calculate the difference between an observed objects's location measured from SAR imagery and its actual, ground-truth, location. The true '*ground truth*' location of an object can be specified in a geocentric rotating coordinate system such as ECEF (Earth-centred, Earth-fixed) as having the coordinate $R_o=[x_o,y_o,z_o]$. The satellite's location in this same reference system changes with time and is given by $R_s(t)=[x_s(t),y_s(t),z_s(t)]$ and has a velocity given by $V_s(t)=[v_{sx}(t),v_{sy}(t),v_{sz}(t)]$. From these, the *slant range* from the satellite to the object as a function of time $s_r(t)$ is given by (1):
+
+$$
+s_r(t) = (x_s(t) - x_o)^2 + (y_s(t)-y_o)^2 + (z_s(t)-z_o)^2 \qquad (1)
+$$
+
+As the satellite transits past the object, the phase of its radar response varies providing the object's signature with a characteristic spatial Doppler frequency shift given by (2):
+
+$$
+f_D(t) = - \frac{2}{\lambda s_r(t)} (\mathbf{R_s(t)} - \mathbf{R_o}) \cdot (\mathbf{V_s(t)} - \mathbf{V_o}) \qquad (2)
+$$
+
+Geometrically, the objects position is found by calculating the intersection between a sphere of radius that corresponds to the range of the object (1), a cone defined by the Doppler frequency at a certain time (2) and the reference ellipsoid (see Fig. 1). There are two locations where all three parametric surface meet - one on the left side of the sensor and one on the right, which is why SAR systems only image on one side.
```

Comment:
[...] the **object's** position is found [...]
[...] three parametric **surfaces** meet [...]